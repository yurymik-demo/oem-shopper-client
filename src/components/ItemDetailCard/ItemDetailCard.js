import React from "react";
import ProgressIndicator from "../ProgressIndicator/ProgressIndicator";

const ItemDetailCard = ({ item }) => {
  if (!item) return <ProgressIndicator />;
  console.log(item);

  const { title } = item;

  return <h2>Item Detail for item {title}</h2>;
};

export default ItemDetailCard;
