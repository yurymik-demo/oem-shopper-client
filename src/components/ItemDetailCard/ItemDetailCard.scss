.item-details-container {
  .card-action {
    min-height: 70px;

    .price {
      margin: 0 10px;
      font-size: x-large;

      @media screen and (max-width: 600px) {
        font-size: large;
      }
    }
  }
}
