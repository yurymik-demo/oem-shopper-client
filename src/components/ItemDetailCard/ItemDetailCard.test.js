import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import ItemDetailCard from "./ItemDetailCard";

it("renders without crashing", () => {
  const mockItem = {
    title: "title",
    description: "item descriotion",
    price: "$123.00"
  };
  const div = document.createElement("div");
  ReactDOM.render(
    <BrowserRouter>
      <ItemDetailCard item={mockItem} />
    </BrowserRouter>,
    div
  );
  expect(div).toMatchSnapshot();
  ReactDOM.unmountComponentAtNode(div);
});
