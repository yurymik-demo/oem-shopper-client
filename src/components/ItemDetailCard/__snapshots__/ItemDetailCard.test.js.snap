// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <h2>
    Item Detail for item 
    title
  </h2>
</div>
`;
